OTTO models update

jhanca-robotecai opened the pull request:
- port https://github.com/RobotecAI/ROSCon2023Demo/pull/255 
- port https://github.com/RobotecAI/ROSCon2023Demo/pull/245 
- port https://github.com/RobotecAI/ROSCon2023Demo/pull/239
- port https://github.com/RobotecAI/ROSCon2023Demo/pull/236
- port https://github.com/RobotecAI/ROSCon2023Demo/pull/230
- update README text
- update README images
- update file permissions

jhanca-robotecai committed:
fix file permissions; fix OTTO1500 model (ROSCon2023Demo#245)

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

jhanca-robotecai committed:
fix materials and textures: ROSCon2023Demo#230 #236 #239 #255

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

jhanca-robotecai committed:
copy 600-1500 prefabs again

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

jhanca-robotecai committed:
fix prefabs after copy

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

jhanca-robotecai committed:
typo in README

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

jhanca-robotecai committed:
fix scripts for lifting platforms

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

jhanca-robotecai committed:
update README images

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

jhanca-robotecai committed:
fix file permissions

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

arturkamieniecki commented:
Otto1500 seems to have problems with steering. This is related with 
![image](https://github.com/RobotecAI/o3de-otto-robots-gem/assets/130671280/e277012f-1a10-45b5-a0f7-8b4a2d802091)
in the wheel entities.

jhanca-robotecai committed:
review: fix OTTO1500 wheel materials

Signed-off-by: Jan Hanca <jan.hanca@robotec.ai>

jhanca-robotecai commented:
Thanks for spotting the problem. I fixed it in both OTTO1500 prefabs. I also rebased `ROSCon2023Demo` branch using this Gem, you might want to check it again: https://github.com/RobotecAI/ROSCon2023Demo/pull/267
